src/Grinder/Commands.fs

```diff
@@ -246,6 +246,16 @@ module Processing =
         else
             CommandNotAllowed
     
+    let private (|UserCanBeBanned|UserCannotBeBanned|) (botSettings: BotSettings, username) =
```

Comment:
Why do you think the active pattern here is better than just a function that returns a `bool`?

Reply:
I wanted to make the code consistent with the similar logic above.